modules/oeFax/contact.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Fax SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2018-2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace Modules\oeFax\Controller;
+
+$ignoreAuth = 0; //todo for debug comment production.
```

Comment:
This thing just makes me nervous; I know it's overkill, but rec either commenting out or removal.